vunit/sim_if/vsim_simulator_mixin.py

```diff
@@ -257,6 +257,14 @@ def _create_gui_script(self, common_file_name, config):
         """
         tcl = f'source "{fix_path(common_file_name)!s}"\n'
         tcl += self._create_user_init_function(config)
+        tcl += 'add_menu "" VUnit\n'
```

Comment:
The add_menu and and add_menuitem commands  are mot supported by Riviera-PRO and Active-HDL. Quickly looking at these tools I don't see anything similar. I suggest that this is limited to ModelSim for now.

Reply:
Oh I did not see that it was used for both ModelSim and Aldec simulators :s
How would you like to procede? Overiding `_create_gui_script` in modelsim.py and calling `super()` in it?
Or adding a variable in the mixin like `extra_gui_commands` and `tcl += f"{extra_gui_commands}\n"` and set in modelsim.py?